layouts/partials/contentNavigation.html

```diff
@@ -0,0 +1,87 @@
+<aside id="aside" class="o-aside o-aside__bottom-sheet">
+  <button
+    class="o-aside__bottom-sheet-header"
+    aria-label="{{ T "contentNavigation.button" }}"
+    aria-controls="bottomSheet-content"
```

Comment:
```suggestion
    aria-controls="bottom-sheet-content"
```

The id of the content element is different.